plugins/Marketplace/stylesheets/marketplace.less

```diff
@@ -90,8 +90,8 @@
             padding-bottom: 0;
             margin-bottom: 10px;
             .more {
-                text-decoration: underline;
                 color: @theme-color-text;
+                white-space: nowrap;
```

Comment:
This is actually a small improvement so that the link content `› more` isn't split onto two lines.